src/cli/getConfiguration.js

```diff
@@ -14,6 +15,7 @@ function createNameAlternatives({ organization, name }) {
     widgetType: `${organization}.${name}`,
     camelCaseName: camelCase(name),
     pascalCaseName: capitalize(camelCase(name)),
+    kebabCaseName: kebabCase(name),
```

Comment:
the name itself should probably already be kebab case, but since you're converting, you should also use it for wigdetType / packageName, right?

Reply:
Okay I just checked and we're using `validate-npm-package-name` which validates `name` for us. It's not necessarily kebab case, but should be fine as a keyword as-is.

https://github.com/algolia/create-instantsearch-app/pull/456/commits/160346348444af8c9bf9bec600b84a51ff573ee6

https://github.com/algolia/create-instantsearch-app/pull/456/commits/b37ad9465c105f34e9f0a923dc9290cdbc3f8a84